crates/anvil/src/lib.rs

```diff
@@ -156,7 +156,14 @@ pub async fn try_spawn(mut config: NodeConfig) -> io::Result<(EthApi, NodeHandle
         let listener = pool.add_ready_listener();
         MiningMode::instant(max_transactions, listener)
     };
-    let miner = Miner::new(mode);
+
+    let miner = match &fork {
+        Some(fork) if !fork.config.read().force_transactions.is_empty() => {
+            let force_transactions = Some(fork.config.read().force_transactions.clone());
+            Miner::new(mode).with_forced_transactions(force_transactions)
+        }
+        _ => Miner::new(mode),
+    };
```

Comment:
this can be converted into a single 

```
Miner::new(mode).with_forced_transactions(config.force_transactions.clone())
```
call

Reply:
updated. sorry if I misinterpreted this comment - I understood you to mean that line 162 assignment to `force_transactions` is unnecessary.

If you were suggesting I turn the whole 160-166 block to a single line, I can do so by either updating the config's `force_transactions` to be an option or to update `with_forced_transactions` to treat an empty vec as None for the purposes of the miner poll logic